### ISSUE TITLE
h20 run infer.py with core dump

### ISSUE THREAD

**Issue by linyubupa** [MoonshotAI/Kimi-Audio#5]
Detected CUDA files, patching ldflags
Emitting ninja build file /data/Kimi-Audio/kimia_infer/models/detokenizer/vocoder/alias_free_activation/cuda/build/build.ninja...
/usr/local/lib/python3.10/dist-packages/torch/utils/cpp_extension.py:1965: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation. 
If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
  warnings.warn(
Building extension module anti_alias_activation_cuda...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
ninja: no work to do.
Loading extension module anti_alias_activation_cuda...
/usr/local/lib/python3.10/dist-packages/torch/nn/utils/weight_norm.py:134: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
Loading '/data/model/moonshotai/Kimi-Audio-7B-Instruct/vocoder/model.pt'
Complete.
using rope base theta = 10000.0, interpolation factor = 1.0
Currently using bfloat16 for PrefixFlowMatchingDetokenizer
model loaded
start generate
Generating tokens:   0%|                                                                            | 1/7356 [00:00<16:01,  7.65it/s]

**Comment by HCRTY** [comment]
我也遇到了一样的问题 有解决吗

**Comment by PeterYoungQaQ** [comment]
我之前用torch2.4.0有这个问题，换成2.5.1+flash_attn-2.7.2.post1没出错

**Comment by HCRTY** [comment]
> 我之前用torch2.4.0有这个问题，换成2.5.1+flash_attn-2.7.2.post1没出错

解决了 感谢感谢

**Comment by f497196689** [comment]
> 我之前用torch2.4.0有这个问题，换成2.5.1+flash_attn-2.7.2.post1没出错

替换了之后还是报错……

**Comment by lengyueyang** [comment]
flash_attn-2.7.2.post1 貌似不用升级，torch torch-audio 升级到 2.5.1 H20 解决 core dump 问题了

**Comment by secslim** [comment]
> > 我之前用torch2.4.0有这个问题，换成2.5.1+flash_attn-2.7.2.post1没出错
> 
> 替换了之后还是报错……

请问解决了吗
